src/Interfaces/AbstractZonotope.jl

```diff
@@ -118,6 +119,25 @@ function generators_fallback(Z::AbstractZonotope{N}) where {N<:Real}
     return FallbackGeneratorIterator(genmat(Z))
 end
 
+"""
+    togrep(Z::AbstractZonotope)
+
+Return a generator representation of a zonotopic set.
+
+### Input
+
+- `Z` -- zonotopic set
+
+### Output
+
+The same set in generator representation.
+This fallback implementation returns a `Zonotope`; in general, other generator
+representations may be returned.
```

Comment:
> ; in general, other generator representations may be returned.

I don't find the part after the semicolon very clear.. can we delete it? Or say that to use other representation, try using `convert` method directly.

Reply:
I wanted to say here: do not rely on the fact that `togrep` *always* returns a `Zonotope`. This is not about `convert`ing.

Reply:
Ah, i see what you mean. The source of my confusion was the part ".. may be returned." (..by whom? Answer: *not* by this function; by other, specific implementations of `togrep` ).

Reply:
Okay, would the following help?
```
This fallback implementation returns a `Zonotope`; however, more specific
implementations may return other generator representations.
```

Reply:
yes